bin/node_modules/nopt/.npmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Why exclude nopt's dependencies when npm pack'ing, and not just exclude nopt itself?